pandas/indexes/base.py

```diff
@@ -2855,11 +2854,18 @@ def join(self, other, how='left', level=None, return_indexers=False):
         how : {'left', 'right', 'inner', 'outer'}
         level : int or level name, default None
         return_indexers : boolean, default False
+        sort : boolean, default False
+
+            .. versionadded:: 0.20.0
```

Comment:
can you add a sentence explaining what sort does (I know its repeatative, but this is a different part of the code)